docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,232 @@
+# Setting Up a Cluster
```

Comment:
Perhaps this should be `Setting Up a Cluster in AWS and Kubernetes` or something? Something to indicate that this is one potential way to set things up. I'm thinking something along the lines of all the k8s guides: http://kubernetes.io/docs/getting-started-guides/gce/